buildstockbatch/aws/aws.py

```diff
@@ -1695,7 +1695,7 @@ def run_job(cls, job_id, bucket, prefix, job_name, region):
         weather_dir = sim_dir / "weather"
         os.makedirs(weather_dir, exist_ok=True)
 
-        epws_to_download = cls.get_epws_to_download(sim_dir, jobs_d)
+        epws_to_download = DockerBatchBase.determine_epws_needed_for_job(sim_dir, jobs_d)
```

Comment:
```suggestion
        epws_to_download = cls.determine_epws_needed_for_job(sim_dir, jobs_d)
```

I'd use `cls` here, which will find the function in the parent class for you while allowing the subclasses to override it if needed.

Same in `gcp.py`.

Reply:
Discussed offline. Making these methods module-level functions instead. They don't use class state (so shouldn't be class functions), and there's no apparent need for them to be overrideable and implementations can easily replace usages at their callsites (so it doesn't need to be a class nor static method).